### DISCUSSION TITLE
My github badges are not presented

### DISCUSSION THREAD

**Discussion by webdev923** [community/community#144270] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've read the github docs about badges, so know how to get pull shark badge.
I made pr several times, but didn't receive any badge yet.
What's wrong?

**Comment by skymagi-d** [comment]
How many pr did you make?

**Comment by pratham-Lashkari** [comment]
To earn the badge, you need a minimum of 2 merged pull requests.

**Comment by webdev923** [comment]
Thank you for your consideration.
I made about 5 pr, but nothing yet.

**Comment by skymagi-d** [comment]
Then did you enable the achievement display setting?

**Comment by webdev923** [comment]
Not sure.
Actually, I'm new here. So I don't know how to do that.

**Comment by skymagi-d** [comment, marked as answer]
No worries.
Go to Settings page on your github.
Check this item and try again.
![image](https://github.com/user-attachments/assets/945d812f-c52c-480b-807f-0d36c99d7836)

**Comment by webdev923** [comment]
Oh, thank you.
